enterprise/backend/src/metabase_enterprise/semantic_search/embedding.clj

```diff
@@ -157,6 +175,31 @@
 (defmethod get-embeddings-batch "ollama" [{:keys [model-name]} text] (ollama-get-embeddings-batch model-name text))
 (defmethod pull-model           "ollama" [{:keys [model-name]}]      (ollama-pull-model model-name))
 
+;;;; AI Service impl
+
+(defn- ai-service-get-embeddings-batch [model-name texts]
+  (try
+    (log/debug "Calling AI Service embeddings API" {:documents (count texts) :tokens (count-tokens-batch texts)})
+    (let [response (metabot-v3.client/generate-embeddings model-name texts)]
```

Comment:
In case it's not clear: kondo error means you need to add `enterprise/metabot-v3` as a module dependency here

https://github.com/metabase/metabase/blob/7bac5b65f92240706a9261575cc64a6d41214191/.clj-kondo/config/modules/config.edn#L1795